apps/modernization-ui/src/pages/patient/profile/identification/IdentificationsTable.tsx

```diff
@@ -75,21 +77,62 @@ export const IdentificationsTable = ({ patient }: Props) => {
     const { changed } = useProfileContext();
 
     const [isActions, setIsActions] = useState<any>(null);
-    const [identifications, setIdentifications] = useState<PatientIdentification[]>([]);
+
+    const asTableBody =
+        (index?: number) =>
+        (identification: Identification): TableBody => ({
+            id: identification.__typename,
+            tableDetails: [
+                {
+                    id: 1,
+                    title: format(identification.asOf, 'MM/dd/yyyy')
+                },
+                { id: 2, title: identification?.type?.description || null },
+                { id: 3, title: identification?.authority?.description || null },
+                { id: 4, title: identification?.value || null },
+                {
+                    id: 5,
+                    title: (
+                        <div className="table-span">
+                            <Button
+                                type="button"
+                                unstyled
+                                disabled={patient?.status !== 'ACTIVE'}
+                                onClick={() => setIsActions(isActions === index ? null : index)}>
+                                <Icon.MoreHoriz className="font-sans-lg" />
+                            </Button>
+                            {isActions === index && (
+                                <ActionState
+                                    handleOutsideClick={() => setIsActions(null)}
+                                    handleAction={(type: string) => {
+                                        tableActionStateAdapter(actions, identification)(type);
+                                        setIsActions(null);
+                                    }}
+                                />
+                            )}
+                        </div>
```

Comment:
I think we'll find with the other related stories that this section here could be a reusable component.

Reply:
I can work with you on getting that component start if you like